packages/desktop-client/src/components/util/GenericInput.jsx

```diff
@@ -209,15 +245,21 @@ export function GenericInput({
           />
         );
       } else {
-        content = (
-          <Input
-            inputRef={inputRef}
-            defaultValue={value || ''}
-            placeholder="nothing"
-            onEnter={e => onChange(e.target.value)}
-            onBlur={e => onChange(e.target.value)}
-          />
-        );
+        switch (type) {
```

Comment:
This switch now has 1 case and a default.  It could be replaced with an if/else (or possibly a single if with an early return/break).